### ISSUE TITLE
Lexicon v3.6 broken

### ISSUE THREAD

**Issue by chaptergy** [certbot/certbot#8826]
## My operating system:
Arch Linux

## I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc):
`pip`

## I ran this command and it produced this output:
```sh
certbot certonly -a dns-njalla --dns-njalla-credentials njalla.ini -d domain.com --dry-run
```

```
Plugins selected: Authenticator dns-njalla, Installer None
Simulating a certificate request for domain.com
Performing the following challenges:
dns-01 challenge for domain.com
Cleaning up challenges
Encountered exception during recovery: TypeError: delete_record() got an unexpected keyword argument 'type'
An unexpected error occurred:
TypeError: create_record() got an unexpected keyword argument 'type'
Please see the logfiles in /var/log/letsencrypt for more details.
```

## Certbot's behavior differed from what I expected because:
It fails when it shouldn't

## Here is a Certbot log showing the issue (if available):
```
2021-05-05 15:34:29,235:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.9/site-packages/certbot/main.py", line 15, in main
    return internal_main.main(cli_args)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/main.py", line 1435, in main
    return config.func(config, plugins)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/main.py", line 1304, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/main.py", line 140, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/client.py", line 444, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/client.py", line 374, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/client.py", line 424, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, best_effort)
  File "/usr/lib/python3.9/site-packages/certbot/_internal/auth_handler.py", line 70, in handle_authorizations
    resps = self.auth.perform(achalls)
  File "/usr/lib/python3.9/site-packages/certbot/plugins/dns_common.py", line 60, in perform
    self._perform(domain, validation_domain_name, validation)
  File "/usr/lib/python3.9/site-packages/certbot_dns_njalla/dns_njalla.py", line 61, in _perform
    self._get_njalla_client().add_txt_record(
  File "/usr/lib/python3.9/site-packages/certbot/plugins/dns_common_lexicon.py", line 48, in add_txt_record
    self.provider.create_record(type='TXT', name=record_name, content=record_content)
TypeError: create_record() got an unexpected keyword argument 'type'
```

## Here is some additional information:
When manually downgrading `dns-lexicon` to v3.5.6 it works.

**Comment by alexzorin** [comment]
This will be fixed by #8819.

**Comment by bmw** [comment]
https://github.com/certbot/certbot/pull/8819 has been merged and will be included in our next release.
